src/main/java/com/github/junit5docker/Port.java

```diff
@@ -1,7 +1,21 @@
 package com.github.junit5docker;
 
+/**
+ * Describes a port binding to send to docker.
+ *
+ * @since 1.0
+ */
 public @interface Port {
+
+    /**
+     * @return the port number exposed on localhost.
+     * This is the port number to use in the test code in order to send data to the container.
```

Comment:
It's not only for sending data :angel: (to `exchange` or to `connect` or something like that).